maestrowf/interfaces/script/localscriptadapter.py

```diff
@@ -89,7 +89,7 @@ def _write_script(self, ws_path, step):
             restart_path = os.path.join(ws_path, rname)
 
             with open(restart_path, "w") as script:
-                script.write(self._exec)
+                script.write("#!"+self._exec)
```

Comment:
So, probably a little bit of a knit pick but I suggest the following:
```
script.write("#!{}".format(self._exec))
```

Reply:
Actually, I take that back. Probably the best way to handle it is:

```
_ = "#!{}\n\n{}\n".format(self._exec, restart)
```

That gets rid of the excess write in the process.

Reply:
Ok I changed it to this though:

"#!{0}\n\n{1}\n".format(self._exec, cmd)
and
"#!{0}\n\n{1}\n".format(self._exec, restart)

Reply:
That's fine -- in this case there's no need to be explicit, but it doesn't hurt anything.

Thanks for updating it.